### ISSUE TITLE
[enchantment] Localisation

### ISSUE THREAD

**Issue by o-murphy** [noXplode/aiogram_calendar#13]
Add please localisation based on chat's locale

**Comment by o-murphy** [comment]
You can use python.locale module with python.calendar

**Comment by noXplode** [comment]
Hello, yes, I`m aware of that
please clarify the purpose of it?
If the goal it to have labels on buttons in language you want, you can pass labels object into calendar class

`selected, date = await SimpleCalendar(labels=labels).process_selection(callback_query, callback_data)`

where labels is a pydantic object of structure:

```
class CalendarLabels(BaseModel):
    "Schema to pass labels for calendar. Can be used to put in different languages"
    days_of_week: conlist(str, max_length=7, min_length=7)
    months: conlist(str, max_length=12, min_length=12)
    cancel_caption: str
    today_caption: str
```

It will allow you to translate captions on Today, Cancel buttons also.

Or do you think of python.locale for changing the date format itself according to locale settings?

**Comment by o-murphy** [comment]
I saw your implementation with the labels object, but you can to do the same more efficiently with using the `python.locale` and `python.calendar` modules, cause using them you don't need to set the labels manualy each time you want to display the calendar in different locale.

Look at the [Telegram Bot API User object here](https://core.telegram.org/bots/api#available-types)
An `User` object has an attribute `language_code` that already provide a locale data for target user, so using that property simplifies to get the `python.calendar` in specified locale in User's `language_code`, and provide a way to do it to all the languages that `python.calendar` supports out of the box

**Comment by o-murphy** [comment]
Simple way is
```python
>>> import locale
>>> locale.setlocale(locale.LC_ALL, 'de_DE')
'de_DE'
>>> import calendar
>>> calendar.month_name[10]
'Oktober'
>>> calendar.day_name[1]
'Dienstag'
```

**Comment by o-murphy** [comment]
One more
```python
import calendar
cal = calendar.LocaleTextCalendar(locale='en_AU.utf8')
print(cal.prmonth(2025, 9))
>>>    September 2025
Mo Tu We Th Fr Sa Su
 1  2  3  4  5  6  7
 8  9 10 11 12 13 14
15 16 17 18 19 20 21
22 23 24 25 26 27 28
29 30

cal = calendar.LocaleTextCalendar(locale='uk_ua')
print(cal.prmonth(2025, 9))
>>>    вересень 2025
Пн Вт Ср Чт Пт Сб Нд
 1  2  3  4  5  6  7
 8  9 10 11 12 13 14
15 16 17 18 19 20 21
22 23 24 25 26 27 28
29 30
```

**Comment by o-murphy** [comment]
[also check this thread](https://stackoverflow.com/questions/13037370/calendar-day-month-names-in-specific-locale)

**Comment by o-murphy** [comment]
### The easyest way

```python
from calendar import different_locale

with different_locale('uk_UA'):
    print(calendar.month_name[10])
>>> жовтень

with different_locale('en_AU'):
    print(calendar.month_name[10])
>>> October
```

**Comment by noXplode** [comment]
Yeah, I'm aware of such implementations.
Will update this week, possible tomorrow, thanks

**Comment by o-murphy** [comment]
> Yeah, I'm aware of such implementations. Will update this week, possible tomorrow, thanks
It works
```python
@dp.message()
async def echo_handler(message: types.Message) -> None:
    try:
        import locale, calendar
        loc = message.from_user.language_code
        alias = locale.locale_alias[loc].split(".")[0]
        with calendar.different_locale(alias):
            await message.answer(calendar.month_name[10])
    except TypeError:
        await message.answer("Err!")
       ```

**Comment by noXplode** [comment]
@o-murphy  can you check https://github.com/noXplode/aiogram_calendar/tree/locale-feature branch
or pr https://github.com/noXplode/aiogram_calendar/pull/14/files

because in replies sometimes user has to be taken from message or from callback made a function for getting locale from user

**Comment by noXplode** [comment]
implemented discussed ability in v0.4

**Comment by o-murphy** [comment]
> implemented discussed ability in v0.4

Looks good

What about "Today" and "Cancel" labels? Have it way to set it directly?
